tracer/tracer.go

```diff
@@ -390,12 +396,46 @@ func initializeMapsAndPrograms(kmod *kallsyms.Module, cfg *Config) (
 	}
 
 	if cfg.OffCPUThreshold > 0 {
-		if err = loadKProbeUnwinders(coll, ebpfProgs, ebpfMaps["kprobe_progs"], tailCallProgs,
+		offCPUProgs := make([]progLoaderHelper, len(tailCallProgs)+2)
+		copy(offCPUProgs, tailCallProgs)
+		offCPUProgs = append(offCPUProgs, []progLoaderHelper{
+			{
+				name:             "finish_task_switch",
+				noTailCallTarget: true,
+				enable:           true,
+			},
+			{
+				name:             "tracepoint__sched_switch",
+				noTailCallTarget: true,
+				enable:           true,
+			},
+		}...)
```

Comment:
nit: The `loadProbeUnwinders` will call `loadProgram` for all individual slice element. So I'm thinking the code would be a bit cleaner if you'd add a separate `if` block earlier to load the `taillCallProgs` first if either of the dependent options is enabled. And then just load a fixed slice here and the following block. This way both feature `if` blocks would be same, and it'd be easier to add future uprobe/kprobe blocks if needed.